src/views/CodeEditor.vue

```diff
@@ -117,8 +121,18 @@ export default {
             });
     },
     methods: {
+        handleCMReady(editor) {
+            this.editor = editor;
+        },
         testCode() {
-            // code to test user's code goes below
+            if (!this.editor) {
+                return;
+            }
+            const userCode = this.editor.getValue();
+            const { tests } = this.challenge;
+            return testRunner(userCode, ["expect(typeof palindrome).to.equal('function')"]).then(result =>
```

Comment:
`challenge.tests` was empty, so I hard coded this one just to be sure it was working.

In the test runner:
- You will have access to the users code as a string as `code`.
- Your tests can use `chai.expect` and `chai.assert`. 
- Async tests will not work correctly at the moment.

In the case of `palindrome`, you will be able to test the function as you would expect to in the context of a unit test, i.e. `expect(palindrome('racecar')).to.be.true` and `expect(palidrome('palidrome')).to.be.false`.